lwt/nbd_lwt_unix.ml

```diff
@@ -21,13 +21,9 @@ type tls_role =
   | TlsClient of Ssl.context
   | TlsServer of Ssl.context
 
-let debug_io = ref false
-
-(* TODO move this function into a library used by this and other executables *)
+(* TODO move this function into a library used by both this and vhd-tool. *)
```

Comment:
I think the what usually happens to TODOs is that we add them and then never do them.